portfolio/src/main/webapp/script.js

```diff
@@ -13,13 +13,36 @@
 // limitations under the License.
 
 $(document).ready(function() {
+    $('#unfilter-button').css("display","none");
```

Comment:
Would recommend that you don't set this initially to invisible from JS, but from CSS instead. That way if there's a problem loading JS, you won't have two buttons.